Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -286,6 +287,7 @@ void WiimoteControllersWidget::LoadSettings(Core::State state)
   const bool running_gc = running && !SConfig::GetInstance().bWii;
   const bool enable_passthrough = m_wiimote_passthrough->isChecked() && !running_gc;
   const bool enable_emu_bt = !m_wiimote_passthrough->isChecked() && !running_gc;
+  const bool allow_netplay = NetPlay::IsNetPlayRunning() ? !running : true;
```

Comment:
This is a bit confusingly named. How about:

```
const bool running_netplay = NetPlay::IsNetPlayRunning() && running;
```

(And then replace uses of `allow_netplay` with `!running_netplay`)

Reply:
Yeah I couldn't think of a good name for this one. That works.